src/cloudflare/internal/vectorize-api.ts

```diff
@@ -18,6 +18,10 @@ enum Operation {
 
 type VectorizeVersion = 'v1' | 'v2';
 
+function toNdJson(arr: object[]) {
+  return arr.map((o) => JSON.stringify(o)).join('\n');
```

Comment:
how big are these arrays expected to be? Asking because using `map` and `join` aren't the fastest ways to do this if the inputs are going to be large (as they end up requiring multiple iterations).

Reply:
The array in this case is as large as the number of vectors being inserted. I'll rewrite it to use a single iteration.